paddle/framework/operator.h

```diff
@@ -83,7 +83,7 @@ class OperatorBase {
     return boost::get<T>(attrs_.at(name));
   }
 
-  virtual std::string DebugString() const;
+  virtual void DebugPrint(std::ostream* os) const;
```

Comment:
agree with @jacquesqiao , overload operator`<<` with 'std::ostream&',  maybe here use same thing will be better?